index.html

```diff
@@ -8,29 +8,194 @@
+        <li class="code-lenguaje-box1">Bootstrap</li>
+        <li class="code-lenguaje-box1">Ruby</li>
+      </ul>
+     
+      <button class="button button_project" >See project</button>
+     </div>
+    </div>
+
+    <div class="deck flex">
+        <img src="./img/photofondo.png" alt="project background" class="imgfondo">
+     <div class="card flex">
+
+      <h3 class="work-titles space">Profesional Art Printing Data</h3>
+      <p class="project-explain space">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+     
+      <ul class="code-lenguaje-use flex space">
+        <li class="code-lenguaje-box1">HTML</li>
+        <li class="code-lenguaje-box1">Bootstrap</li>
+        <li class="code-lenguaje-box1">Ruby</li>
+      </ul>
+     
+      <button class="button button_project" >See project</button>
+     </div>
+    </div>
+
+    <div class="deck flex">
+        <img src="./img/photofondo.png" alt="project background" class="imgfondo">
+     <div class="card flex">
+
+      <h3 class="work-titles space">Profesional Art Printing Data</h3>
+      <p class="project-explain space">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+     
+      <ul class="code-lenguaje-use flex space">
+        <li class="code-lenguaje-box1">HTML</li>
+        <li class="code-lenguaje-box1">Bootstrap</li>
+        <li class="code-lenguaje-box1">Ruby</li>
+      </ul>
+     
+      <button class="button button_project" >See project</button>
+     </div>
+    </div>
+
+    <div class="deck flex">
+        <img src="./img/photofondo.png" alt="project background" class="imgfondo">
+     <div class="card flex">
+
+      <h3 class="work-titles space">Profesional Art Printing Data</h3>
+      <p class="project-explain space">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+     
+      <ul class="code-lenguaje-use flex space">
+        <li class="code-lenguaje-box1">HTML</li>
+        <li class="code-lenguaje-box1">Bootstrap</li>
+        <li class="code-lenguaje-box1">Ruby</li>
+      </ul>
+     
+      <button class="button button_project" >See project</button>
+     </div>
+    </div>
+</section>
+
+ <section class="about_me_page flex">
+
+     <h4 id="aboutme-title">About me</h1>
+     <p id="aboutme-text">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+
+      <button class="button">Get my resume</button>
+
+     </section>
+
+ <section class="knowledge_page">
+  <img src="./img/Devider.png" alt="division line" class="devider-line">
+
+ <h5 class="skill_mention"><img class="orange_figure" src="./img/diamond-orange.png" alt="orange figure">Lenguages</h5>
```

Comment:
- **[OPTIONAL]** As suggested by the previous reviewer please fix this typo here:
```suggestion
 <h5 class="skill_mention"><img class="orange_figure" src="./img/diamond-orange.png" alt="orange figure">Languages</h5>
```